### DISCUSSION TITLE
import sklearn in python 3.11.1

### DISCUSSION THREAD

**Discussion by ZahidUllah1992** [AammarTufail/python_ka_chilla2023#70] [General]
I am trying to import the sklearn datasets in python in 3.11.1, however works on python 3.9 version
from sklearn.linear_model import LinearRegression
 
However, I am having the following error,
ImportError: DLL load failed while importing _iterative: %1 is not a valid Win32 application.

After searching, I found that scikit-image is not available for python 3.11.1.
I installed pip install scikit-image

still it is not working and showing that install pyproject.toml is required

Anyone having the same issue, please let me know

**Comment by usamabajwa86** [comment]
i am using scikit-learn in python 3.11
run these two commands

pip install scikit-learn
!pip install scikit-learn

**Reply by ZahidUllah1992** [reply]
i had installed all these, i think this is some issue relevant to wheel, maybe @AammarTufail  knows about it, I have accessed several github forums but unable to address it. The problem arosed when I tried to run the fromsklear... in python 3.11, when I changed the python from anaconda 3.9 it start to run.

**Reply by ZahidUllah1992** [reply]
still ImportError: DLL load failed while importing _iterative: %1 is not a valid Win32 application.

**Reply by usamabajwa86** [reply]
i had same issue, but resolved today with multiple commands

**Comment by ZahidUllah1992** [comment]
Ok i had already run. Let me recheck

Zahid
________________________________

**Comment by ZahidUllah1992** [comment]
Please share the commanda

Zahid
________________________________
